src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -95,24 +106,21 @@ export async function getCowSwapTradeQuote(
 
   const { data } = maybeQuoteResponse.unwrap()
 
-  const {
-    feeAmount: feeAmountInSellTokenCryptoBaseUnit,
-    buyAmount: buyAmountAfterFeesCryptoBaseUnit,
-  } = data.quote
+  const { feeAmount: feeAmountInSellTokenCryptoBaseUnit } = data.quote
```

Comment:
nitpick - could just use `getValuesFromQuoteResponse` to pass back this one as well so it is all happening in the same place

Reply:
I mean we could but since we're not really mutating anything doesn't make sense - and this helper is really only used in one place, to hide the logic away so it having a nice interface isn't a big deal IMO?

Reply:
yeah not a big deal